docs/api/algorand/one-to-many-transfer.mdx

```diff
@@ -0,0 +1,94 @@
+# OneToManyTransfer
+
+Send funds from a [DelegatedWallet](#TODO), to multiple other (regular or delegated) wallets.
+
+## Request Parameters
+
+|     Name    |                    Type                   | Description |
+| ----------- | ----------------------------------------- | ----------- | 
+| `assetId`   | `string`                                  | The ID of the DeCash asset that is being sent. |
+| `sender`    | `string`                                  | The address of the account that is sending the funds. Note that this account must have enough funds to cover both all the `amount`s, and the [fee](#TODO)s. |
```

Comment:
The transaction fee? Is it a link to concepts?

Reply:
Yep, I think it makes sense to somewhere talk about how much is spent on fees